sklearn/manifold/_t_sne.py

```diff
@@ -597,6 +598,14 @@ class TSNE(BaseEstimator):
 
         .. versionadded:: 0.22
 
+    square_distance : {True, 'legacy', 'warn'}, default='warn'
```

Comment:
Forgot that in my last review. We don't put 'warn' in the docstrings.

```suggestion
    square_distance : {True, 'legacy'}, default='legacy'
```

Reply:
Thanks for the additional comments! :)